docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -53,8 +53,168 @@ Intel® Core™ i3-8121U Processor
+|5|1|1|1|1|1|256|76|76|76|76|51|25|
+|5|1|1|1|1|1|384|76|76|76|76|51|25|
+|6|1|1|1|1|1|128|64|64|64|64|42|21|
+|6|1|1|1|1|1|256|64|64|64|64|42|21|
+|6|1|1|1|1|1|384|64|64|64|64|42|21|
+|7|1|1|1|1|1|128|54|54|54|54|36|
+|7|1|1|1|1|1|256|54|54|54|54|36|
+|7|1|1|1|1|1|384|54|54|54|54|36|
+
+<b>Table of Maximum Input Tensor Widths (W) vs. Rest of Parameters</b><br>
+(Input and Kernel Precision: 1 bytes)<br>
+
+|KH|KW|PH|PW|SH|SW|H|W<br>Ci=8<br>Co=256|W<br>Ci=16<br>Co=256|W<br>Ci=32<br>Co=256|W<br>Ci=64<br>Co=256|W<br>Ci=128<br>Co=256|W<br>Ci=256<br>Co=256|W<br>Ci=384<br>Co=256|
+|:--|:--|:--|:--|:--|:--|:--|:--|:--|:--|:--|:--|:--|:--|
+|1|1|1|1|1|1|128|240|240|240|240|240|240|240|
+|1|1|1|1|1|1|256|240|240|240|240|240|240|170|
+|1|1|1|1|1|1|384|240|240|240|240|240|170|113|
+|1|2|1|1|1|1|128|240|240|240|240|240|240|240|
+|1|2|1|1|1|1|256|240|240|240|240|240|240|170|
+|1|2|1|1|1|1|384|240|240|240|240|240|170|113|
+|1|3|1|1|1|1|128|240|240|240|240|240|
+|1|3|1|1|1|1|256|240|240|240|240|240|
+|1|3|1|1|1|1|384|240|240|240|240|240|
+|2|1|1|1|1|1|128|192|192|192|192|192|192|192|
+|2|1|1|1|1|1|256|192|192|192|192|192|192|170|
+|2|1|1|1|1|1|384|192|192|192|192|192|170|113|
+|2|2|1|1|1|1|128|193|193|193|193|193|193|129|
+|2|2|1|1|1|1|256|193|193|193|193|193|193|129|
+|2|2|1|1|1|1|384|193|193|193|193|193|170|113|
+|3|1|1|1|1|1|128|128|128|128|128|128|128|85|
+|3|1|1|1|1|1|256|128|128|128|128|128|128|85|
+|3|1|1|1|1|1|384|128|128|128|128|128|128|85|
+|3|3|1|1|1|1|128|130|130|130|130|87|
+|3|3|1|1|1|1|256|130|130|130|130|87|
+|3|3|1|1|1|1|384|130|130|130|130|87|
+|4|1|1|1|1|1|128|96|96|96|96|96|96|64|
+|4|1|1|1|1|1|256|96|96|96|96|96|96|64|
+|4|1|1|1|1|1|384|96|96|96|96|96|96|64|
+|5|1|1|1|1|1|128|76|76|76|76|76|51|51|
+|5|1|1|1|1|1|256|76|76|76|76|76|51|51|
+|5|1|1|1|1|1|384|76|76|76|76|76|51|51|
+|6|1|1|1|1|1|128|64|64|64|64|64|42|21|
+|6|1|1|1|1|1|256|64|64|64|64|64|42|21|
+|6|1|1|1|1|1|384|64|64|64|64|64|42|21|
+|7|1|1|1|1|1|128|54|54|54|54|54|36|18|
+|7|1|1|1|1|1|256|54|54|54|54|54|36|18|
+|7|1|1|1|1|1|384|54|54|54|54|54|36|18|
+
+
+> **NOTE**:  The above limitations only apply to the new hardware 2D convolution operation.  When possible, the Intel® GNA plugin graph compiler flattens 2D convolutions so that the second generation Intel® GNA 1D convolution operations (without these limitations) may be used.  The plugin will also flatten 2D convolutions regardless of the sizes  if GNA 2.0 compilation target is selected (see below).
+
+### Intel® GNA Forward and Backward Compatibility
+
+In the general case, there is no guarantee that a model compiled for GNA 2.0 will run on GNA 3.0, or vice versa.
+
+However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the Models and Layers Limitations section).
```

Comment:
I don't see the Models and Layers Limitations section in this document.  Please provide a link.